.circleci/config.yml

```diff
@@ -159,7 +159,9 @@ jobs:
             cp -a keys.example keys
             cp -a certs.example certs
             cp pwned_passwords/pwned_passwords.txt.sample pwned_passwords/pwned_passwords.txt
-            bundle exec rake db:setup --trace
+            # bundle exec rake db:setup --trace
+            git checkout master -- db/schema.rb
```

Comment:
This will need to be updated to `main` when we make the switch

Reply:
This doesn't make sense to me, why don't we run the migrations that are on the branch being tested? If every CI build just checks out our default branch, we won't be able to see changes until after they're merged?

Reply:
We are - this command checks out **only** the `db/schema.rb` file, which we then load (`db:schema:load`) before running the migrations that exist in the branch (`db:migrate`). Does that make sense?

Reply:
What if we just updated CI to run migrations from scratch?

Reply:
I'm curious about this as well. It's slower and I think there are some unsafe migrations in the past that would fail now. Is that the primary obstacle?

Reply:
Yes - less the speed and more the fact that we'd have to edit a number of old migrations to make them safe.

Reply:
(As I think about it more it occurs to me that we can just wrap them in `safety_assured` and call it a day - what do you think?)

Reply:
It looks like there's a `StrongMigrations.start_after =` option where it will only run strong migrations checks against migrations after a specific migration: https://github.com/ankane/strong_migrations/blob/fff64d21f9f6294c19920e88bf3664d962228e91/README.md#existing-migrations

Reply:
Oooh great find, let me do some Git digging to figure out when we added it

Reply:
Looks like it works! I'm just (for my own curiosity's sake) checking that the migration passes with a later version of Postgres, then I'll clean up the commits and request a new review. Thanks!